### PR TITLE
Create prohibited-hardware-and-software.md

### PR DESCRIPTION
**Pull request by sammydelao**
----
:books: Documentation preview :books:: https://civicactions-handbook--1222.org.readthedocs.build/en/1222/

### CODE REVIEW

File: company-policies/prohibited-hardware-and-software.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Prohibited Hardware and Software
+
+## Summary
+
+CivicActions has established a list of hardware and software that is prohibited to use for CivicActions activities, to meet compliance with Federal Acquisition Regulation (FAR) requirements for work on federal contracts.
+
+## Requirements
+
+The following is a list of of hardware and software that is prohibited to use for based on FAR requirements:
```
Comment (dmundra) [inline company-policies/prohibited-hardware-and-software.md]: @sammydelao small typo here double 'of'

**Review by dmundra — CHANGES_REQUESTED** [review]
Left you a comment on the other pull request well to bring it into this one. Our automatic checks found one small typo. Please review that in the 'Files changed' tab.

**Review by dmundra — APPROVED** [review]
Thank you @sammydelao and @grugnog

**Review by grugnog — APPROVED** [review]
@sammydelao this looks good to me! I think we have a reference to this in our onboarding, but that would be good to check that if you didn't already :)

**Pull request merged**